src/AyonCppApi/AyonCppApi.cpp

```diff
@@ -78,11 +209,17 @@ std::unordered_map<std::string, std::string>*
 AyonApi::getSiteRoots() {
     m_Log->info(m_Log->key("AyonApi"), "AyonApi::getSiteRoots()");
     if (m_siteRoots.size() < 1) {
-        httplib::Headers headers = {{"X-ayon-site-id", m_siteId}};
         nlohmann::json response
-            = GET(std::make_shared<std::string>("/api/projects/" + m_ayonProjectName + "/siteRoots"),
-                  std::make_shared<httplib::Headers>(headers), 200);
-        m_siteRoots = response;
+            = GET(std::make_shared<std::string>("/api/projects/" + m_ayonProjectName + "/siteRoots?platform=linux"),
+                  std::make_shared<httplib::Headers>(m_headers), 200);
```

Comment:
```suggestion
    #ifdef _WIN32
       response = GET(std::make_shared<std::string>("/api/projects/" + m_ayonProjectName + "/siteRoots?platform=windows"), std::make_shared<httplib::Headers>(m_headers), 200);
    #else
       response = GET(std::make_shared<std::string>("/api/projects/" + m_ayonProjectName + "/siteRoots?platform=linux"), std::make_shared<httplib::Headers>(m_headers), 200);
    #endif

```



would be best to set the windows Linux etc options at compile time 
cheap solution via ifdef ? like the ones you did atop ? 

https://docs.ayon.dev/api#tag/Projects/operation/get_project_site_roots